pages/[slug]/itinerary.js

```diff
@@ -121,13 +142,20 @@ export default function Itinerary({ trips, sights, setSights }) {
           ))}
         </StyledItinerary>
       </Scrollbox>
-      <StyledFooter>
-        <StyledLink href="/">Home</StyledLink>
-      </StyledFooter>
+      <Footer buttonlink={`/${currentTrip.slug}`} buttontext="Overview" />
     </>
   );
 }
 
+const StyledLottie = styled(Lottie)`
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 50vw;
+  height: 50vw;
+`;
+
```

Comment:
Do you use this styled component? If not it should be removed. :)
```suggestion

```

Reply:
Yep, but it's not in the "normal" return. It's all the way at the top and used as a placeholder, until currentTrip is available.